src/entities/permit2/domain.ts

```diff
@@ -0,0 +1,20 @@
+import { Address, TypedData, TypedDataDomain } from 'viem';
+
+const PERMIT2_DOMAIN_NAME = 'Permit2';
+
+export function permit2Domain(
+    permit2Address: Address,
+    chainId: number,
+): TypedDataDomain {
+    return {
+        name: PERMIT2_DOMAIN_NAME,
+        chainId,
+        verifyingContract: permit2Address,
+    };
+}
+
+export type PermitData = {
+    domain: TypedDataDomain;
+    types: TypedData;
+    values: any;
+};
```

Comment:
Is this type declaration being used?

Reply:
Apparently not 😅 I adapted the implementation from another repo and I forgot to delete it.
Will remove 👍